### ISSUE TITLE
access the on boarding flow multiple times

### ISSUE THREAD

**Issue by ring-ring-ring** [minvws/nl-kat-coordination#913]
The on boarding flow has grown in to a more generic 'wizard' to add objects and request reports. Perhaps it is useful to access it not just once after the first login but more often, as it might help users to make the most of the system.

**Comment by underdarknl** [comment]
Yes. The idea is to add a 'report' button to the menu-bar, from where you can choose a specific report.
If then you choose (for example) a dns report, and no OOI's exist for the needed type (url in the case of dns resports), you'd expect the same dialog to be presented as you would see in the onboarding flow. It should also take into account indemnification levels, and offer those steps.
Missing required boefjes/plugins/bits should offer those steps.

For every report, a set of allowed input OOI's, required plugins should be enough to check against.
